worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/PartitionDataWriter.java

```diff
@@ -566,7 +566,7 @@ protected void waitOnNoPending(AtomicInteger counter) throws IOException {
       waitTime -= WAIT_INTERVAL_MS;
     }
     if (counter.get() > 0) {
-      IOException ioe = new IOException("Wait pending actions timeout.");
+      IOException ioe = new IOException("Wait pending actions timeout, Counter: " + counter.get());
```

Comment:
```suggestion
      IOException ioe = new IOException("Wait pending actions timeout, pending flushes count: " + counter.get());
```

Reply:
Pending actions have two types: numPendingWrites and numPendingFlushes, So I use counter here.